### DISCUSSION TITLE
create a recipe to setup the environment

### DISCUSSION THREAD

**Discussion by BWhannou** [ML-for-B-E/decision_tree_course#2] [Ideas]
@akouete-kpakpo In your README.md you presented how to setup the virtual environment and run the notebook with that venv. wouldn't be better to create a command in a make file so that participant could run just one command? 

my idea is: let's try to encapsulate everything that we don't want them to understand necessarly.

what do you think?

**Comment by akouete-kpakpo** [comment]
@BWhannou good idea. But `make` is difficult to setup on Windows machines. 

If we assume a Linux environment for everyone then it we can do it. Personally I use windows WSL2 with ubuntu 20.04 LTS on my laptop (a Linux Virtual Machine inside my Windows laptop). 

In any case I will implement a recipe in Makefile to setup the venv in the CI (I will assume Ubuntu environment). Cf https://github.com/ML-for-B-E/.github/issues/2

**Reply by BWhannou** [reply]
True! you need the recipe so that the CI can run the code in the proper environment, understood. thanks @akouete-kpakpo !
